recipes/boltons/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "16.2.1" %}
+
+package:
+    name: boltons
+    version: '{{ version }}'
+
+source:
+    git_url: https://github.com/mahmoud/boltons.git
+    git_rev: '{{ version }}'
+
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+    run:
+        - python
+
+test:
+    requires:
+        - pytest
+    commands:
+        - python -c "from platform import system as s; import sys; sys.exit(int(s()!='Windows'))" || py.test --doctest-modules $SRC_DIR/boltons $SRC_DIR/tests
+        - python -c "from platform import system as s; import sys; sys.exit(int(s()=='Windows'))" || py.test --doctest-modules %SRC_DIR%/boltons %SRC_DIR%/tests
```

Comment:
If I read this right: https://github.com/conda/conda-build/blob/dd7e325f6882765b12c12f9b0bb1b1c294385084/conda_build/jinja_context.py#L58

then `SRC_DIR` is part of the jinja context (via `get_environ()`)